src/skia/RuntimeEffect.cpp

```diff
@@ -0,0 +1,197 @@
+#include <stdexcept>
+#include "common.h"
+#include <include/effects/SkRuntimeEffect.h>
+//#include <include/core/SkM44.h> // defines SkV3, SkV4 ; M44 used in Matrix/Canvas ; Revisit.
+#include <pybind11/stl_bind.h>
+
+PYBIND11_MAKE_OPAQUE(std::vector<SkRuntimeEffect::ChildPtr>)
+
+void initRuntimeEffect(py::module &m) {
+py::class_<SkRuntimeEffect, sk_sp<SkRuntimeEffect>, SkRefCnt> runtime_effect(m, "RuntimeEffect");
+
+py::class_<SkRuntimeEffect::ChildPtr> runtime_effect_childptr(m, "RuntimeEffectChildPtr");
+
+py::bind_vector<std::vector<SkRuntimeEffect::ChildPtr>>(m, "VectorSkRuntimeEffectChildPtr");
+py::class_<SkSpan<const SkRuntimeEffect::ChildPtr>> span_runtime_effect_childptr(m, "SpanRuntimeEffectChildPtr");
+
+py::class_<SkRuntimeEffectBuilder> runtime_effect_builder(m, "RuntimeEffectBuilder");
+
+py::class_<SkV3>(m, "V3")
```

Comment:
Is supporting an implicit conversion from `list` or `tuple` helpful for users, like `SkColor4f`?

Reply:
Yes, I think v4 those are almost like color4f's except it can take half-precisions. It is all new to but I can check

Reply:
I added implicit conversion for `tuple` and thought it didn't work, so I overloaded `setUniform` to take `tuple` and found I was passing a `list` (always a bit confused about which is which myself), I like the list syntax a bit better; but now both works, just via different mechanisms!

Am a bit curious how python chooses, if tuple is implicit convertible to both V3 and V4 (depending on length = 3 or 4).